contracts/rollupReddit.sol

```diff
@@ -157,13 +160,14 @@ contract RollupReddit {
             bool
         )
     {
-        Types.Transaction memory _tx = RollupUtils.TxFromBytes(txBytes);
-        _tx.signature = sig;
+        bytes memory txs = RollupUtils.CompressTransferFromEncoded(txBytes);
+        // Validate ECDSA sig
         return
             transfer.processTx(
```

Comment:
the offchain `processTx` use utils to convert encoded bytes of `Types.Transaction` to txs and index 0, then pass it to the onchain `processTx` for consistency of behavior on chain.